app/components/activity/index.ts

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+import template from './template.html!text';
+
+export default function(app) {
+  app.component('activity', {
+    bindings: {
+      discussion: '<',
+      document: '<',
+      user: '<',
+    },
+    controller: [
+    '$http', 'config', 'notificationService',
+      function($http, config, notificationService) {
+        const ctrl = this;
+        ctrl.$onChanges = function(changesObj) {
+          let personId;
+          if (ctrl.user) {
+            personId = ctrl.user.id;
+          }
```

Comment:
I think we should do something like this:
```javascript
if (!ctrl.user) {
  ctrl.activities = undefined;
  return;
}
const personId = ctrl.user.id;
```
Otherwise, the HTTP request is sent with an undefined `personId`.